GETTING-STARTED.md

```diff
@@ -0,0 +1,60 @@
+# Getting Started
+
+## Setup and Installation
+
+### Prerequisites
+
+In order to use **Django Deploy**, you must first install the following dependencies:
+- [Python](https://www.python.org/downloads/) 3.5 or higher
+- [virtualenv](https://virtualenv.pypa.io/en/stable/installation/)
+- [Docker](https://docs.docker.com/install/overview/) (any edition)
+- [Google Cloud SDK](https://cloud.google.com/sdk/docs/quickstarts)
+- [kubectl](https://cloud.google.com/kubernetes-engine/docs/quickstart) (can be
```

Comment:
We do not need kubectl anymore.

Reply:
Ahhh...I screwed up this PR - this file shouldn't be included. I'll abandon this one and resubmit it.